cmd/pinniped/cmd/kubeconfig.go

```diff
@@ -151,14 +153,16 @@ func kubeconfigCommand(deps kubeconfigDeps) *cobra.Command {
 	f.StringVarP(&flags.outputPath, "output", "o", "", "Output file path (default: stdout)")
 	f.StringVar(&flags.generatedNameSuffix, "generated-name-suffix", "-pinniped", "Suffix to append to generated cluster, context, user kubeconfig entries")
 	f.StringVar(&flags.credentialCachePath, "credential-cache", "", "Path to cluster-specific credentials cache")
+	f.StringVar(&flags.commandExePath, "command-exe-path", "", "Path to use as the command for the client-go credential plugin, usually the pinniped CLI")
```

Comment:
"exe" is MS Windows terminology, right? Could we drop that part?

If we called this flag `--command-path` it wouldn't really indicate which command you're talking about. Maybe we can be more specific.

How about something like `--pinniped-cli-path` instead? This needs to be the path to the Pinniped CLI in practice because it needs to be able to accept all the custom arguments that we are going to specify in the resulting kubeconfig.

Could we also describe in the help message for this flag what the default behavior will be? Maybe something like:

`Full path or executable name for the Pinniped CLI binary to be embedded in the resulting kubeconfig output (default: basename of the binary used to execute this command with ".exe" removed)`

Reply:
I avoided the name `pinniped` because technically this gives away the flexibility to substitute in the binary of the end user's choice. But at the same time `command-*` doesn't really say what the thing is used for.

Reply:
Let's do `--pinniped-cli-path`.